README.md

```diff
@@ -89,4 +84,22 @@ mkvirtualenv -p /usr/local/bin/python3 notifications-template-preview
 pip install -r requirements.txt
 ```
 
-Then to run, activate the virtualenv and call `./scripts/run_app.sh 6013`
+Then create a `version.py` file under the app folder, you can copy `version.py.dist` to `version.py` to get it running.
+
+You will also need to set VCAP_SERVICES (extracted from Makefile) -
+
+```shell
+export VCAP_SERVICES='{"user-provided":[{"credentials":{"api_host": "some_domain","api_key":"my-secret-key"},"label":"user-provided","name":"notify-template-preview","syslog_drain_url":"","tags":[],"volume_mounts":[]}]}'
+```
+
+Then to run, activate the virtualenv and call the run app script -
+
+```shell
+./scripts/run_app.sh 6013
+```
+
+Thereafter activate the virtualenv prior to executing the run script -
+
+```shell
+workon notifications-template-preview
+```
```

Comment:
in fact i think all three of these code blocks should be combined since it doesn't make sense to do any one step without the others